front/app/modules/commercial/insights/admin/containers/Reports/Charts/CustomFieldGraph.tsx

```diff
@@ -193,6 +195,48 @@ const CustomFieldsGraph = ({
       ? customFieldEndpoints[code as TAllowedCode].xlsxEndpoint
       : usersByCustomFieldXlsxEndpoint(customField.id);
 
+  // If there is no code, this is not a built-in field
+  if (!customField.attributes.code) {
+    if (customField.attributes.input_type === 'checkbox') {
+      return (
+        <Box width="50%">
+          <Box width="1340px">
```

Comment:
These chart components don't act the exact same way as the existing CustomFieldGraph does on this page, so the width styling is a bit odd here but it works :thinking:

Reply:
Yeah but it's probably fine, I don't think we should spend too much time on this since we want to get rid of these reports anyway